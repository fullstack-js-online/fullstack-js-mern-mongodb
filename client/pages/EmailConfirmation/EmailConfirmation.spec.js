import React from 'react'
import { mount } from 'enzyme'

import { EmailConfirmationPage } from './'

describe('The EmailConfirmation page', () => {
  const props = {
    match: {
      params: {
        token: ''
      }
    },
    dispatch: () => new Promise(() => {})
  }

  it('should match snapshot', () => {
    expect(mount(<EmailConfirmationPage {...props} />)).toMatchSnapshot()
  })
})
