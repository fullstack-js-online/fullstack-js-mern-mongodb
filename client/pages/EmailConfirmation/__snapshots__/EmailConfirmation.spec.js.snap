// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The EmailConfirmation page should match snapshot 1`] = `
<EmailConfirmationPage
  dispatch={[Function]}
  match={
    Object {
      "params": Object {
        "token": "",
      },
    }
  }
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <SideEffect(NullComponent)
      defer={true}
      encodeSpecialCharacters={true}
      title="Email confirmation"
      titleAttributes={Object {}}
    >
      <NullComponent
        defer={true}
        encodeSpecialCharacters={true}
        title="Email confirmation"
        titleAttributes={Object {}}
      />
    </SideEffect(NullComponent)>
  </HelmetWrapper>
  <EmailConfirmationLoader>
    <h1
      className="text-center"
    >
      Confirming Email ...
    </h1>
  </EmailConfirmationLoader>
</EmailConfirmationPage>
`;
